README.md

```diff
@@ -219,6 +237,10 @@ dbname.
 created in this instance. If not specified, the tool automatically determines an
 appropriate instance using gcloud.
 
+`-driver` Specifies the driver to use for schema and data conversion. Supported drivers
```

Comment:
By default, the driver is `pgdump`. Please update this statement accordingly.

Reply:
As discussed in previous comment, currently there is no default driver and specifying a driver is must. Let me know what do you think and then I will update accordingly.

Reply:
And remove the "It is mandatory to specify a driver."

Reply:
Yes, sorry, we should have discussed this in a bit more detail.

https://github.com/cloudspannerecosystem/harbourbridge/pull/63 added the default.

What happened is that #62 broke the integration test. Hengfeng and I discussed various options for fixing. In the end we decided to have pgdump as the default to preserve the original harbourbridge behaviour i.e. harbourbridge just works out-of-the-box with pg_dump, no flags, no args, no configs.

This is done by initializing driverName to "pgdump" (line 74 of main.go).

Reply:
Ah! ok. I didn't notice that change. Thanks for the update. 
Maybe it's worth adding the point (default driver is PGDUMP) in README. I will update it.